main.cpp

```diff
@@ -175,20 +203,23 @@ struct Hotel
     Guest sally;
 };
 
-/*
-Thing 2) Printer
-5 properties:
-    1) Ink levels 
-    2) Electrcity consumption rate
-    3) Paper level
-    4) Maximum resolution
-    5) Brand name
-3 things it can do:
-    1) Print documents
-    2) Load print jobs
-    3) Scan documents
- */
+void Hotel::bookGuests(Hotel::Guest guest)
+{
+    float totalCost = guest.costPerNight * guest.lengthOfStay;
+    std::cout << "Total cost is: " << totalCost;
 
+}
+
+void Hotel::cleanRoom(int roomNumber)
+{
+    std::cout << "Hi, would you like room " << roomNumber <<"cleaned?";
+}
+
+void Hotel::valetCar(Hotel::Guest guest, int parkingSpot)
+{
+    guest.costPerNight += 20.f;
+    std::cout << "Delier car to parking spot " << parkingSpot;
```

Comment:
This is a common issue with new programmers or programmers who are coming from garbage-collected languages (python, C#).
When you pass this type into this function, the instance that is used inside this function is NOT the same instance that was passed in.
That means any changes you make to the instance being used INSIDE the function are NOT kept when this function ends.
the C++ concept of References and Pointers has not been covered at this point in the course.  If you wanted this to work as you are most likely expecting it to work, you would need to use references or pointers.
I don't let people use coding concepts in projects until after they've been covered in the course.
You don't need to revise these function parameters.
I just want you to be aware of the potential issue.